src/main/java/appeng/me/cache/NetworkMonitor.java

```diff
@@ -21,99 +21,252 @@
 
 import java.util.Collection;
 import java.util.Deque;
+import java.util.HashMap;
 import java.util.Iterator;
 import java.util.LinkedList;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.LinkedList'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.orod.org/coding_rules#rule_key=squid%3AUselessImportCheck)